code/modules/mapping/mapping_helpers.dm

```diff
@@ -1137,3 +1137,85 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	var/turf/open/floor/floor = get_turf(src)
 	floor.burn_tile()
 	qdel(src)
+
+//machinery helpers
```

Comment:
a bit more descriptive DMDoc (with `///` comments) would be well appreciated for statistical purposes at the very least.